### PR TITLE
fix(core): allow usage of Pebble verbatim tag in our recursive renderer

### LINKED ISSUE #1244: Variables rendering escaping
We have some documentation to [allow espace](http://localhost:3001/docs/developer-guide/variables/tag/verbatim) of `{{` if you have a templating engine behind the scene on the properties.

This one don't work since we have a recursive rendering that would inhibit the [variable rendering](https://github.com/kestra-io/kestra/blob/fea0e56715b8c76b67d319a479344592c067771e/core/src/main/java/io/kestra/core/runners/VariableRenderer.java#L91). 

We need to introduce a proper ways to allow escaping `{{`
We could choose: 
- `{ { Location } }`
- `{% raw %}{{ Location }}{% endraw %}`: don't think we are able since the verbatim tag is implemented on the raw parser
- `{\{ Location }}`

### PR DESCRIPTION
**Pull request by loicmathieu**
close #1244

After investigation, due to the fact that we render a template recursively, we cannot use an operator like `\{{` nor a function, nor a filter :(

I ended up using a hack to stop the recursivity if using the verbatim tag that is the tag that should have been working for this. Inline verbatim is not possible.

More advanced solutions (function, tag, filter) would need to hook into the template engine itself as at the render level we only have access to the template and its result.

Another solution would be to stop inline rendering and provide a `render` function so users opt-in for recursive rendering.

I'm not very happy with the solution so if someone has a better idea I'm open to suggestions.

### CODE REVIEW

File: core/src/main/java/io/kestra/core/runners/VariableRenderer.java [hunk 1]
```diff
@@ -128,14 +128,17 @@ public String recursiveRender(String inline, Map<String, Object> variables) thro
...
         }
+        while(!isSame && !isVerbatim(inline));
 
         return current;
     }
 
-    public IllegalVariableEvaluationException properPebbleException(PebbleException e) {
-        if (e instanceof AttributeNotFoundException) {
-            AttributeNotFoundException current = (AttributeNotFoundException) e;
+    private boolean isVerbatim(String inline) {
```
Comment (tchiotludo) [inline core/src/main/java/io/kestra/core/runners/VariableRenderer.java:136]: will only work with a raw starting with `{%verbatim` and will failed with `content{%verbatim` that is a must have.

**Review by tchiotludo — CHANGES_REQUESTED** [review]
verbatim could work for me, and we need to find a syntax that is not `{{` or `{%`, and don't think we could trick on that since even the author of pebble had to trick

**Comment by loicmathieu** [comment]
@tchiotludo I had a new lightweight idea to replace `\{{` and `\}}`  with an escape sequence and replace back at the end.
It's an easy solution that at least allow to display `{{` without too many modification.
We can use another replace sequence that the one I propose here, the PR is just a demo to know if you think it would be acceptable.

**Comment by tchiotludo** [comment]
close in favor of this one https://github.com/kestra-io/kestra/pull/1316

**Pull request closed without merging**